feedback/views.py

```diff
@@ -780,6 +784,65 @@ def get_form_kwargs(self):
         return super().get_form_kwargs()
 
 
+class ImportTagsView(ManageCourseMixin, FormView):
+    template_name = "feedback_tags/import_tags.html"
+    form_class = ImportTagsForm
+
+    def get_course_options(self):
```

Comment:
I changed this method so that only courses visible to the teacher are in the dropdown menu, as the main use case is to import tags from the previous course instance. Also, all the preview tables are rendered in advance (and marked as hidden), so this change saves a lot of unnecessary html rendering.